README.md

```diff
@@ -1,7 +1,5 @@
-# curriculum-databases-projects-template
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+![](https://img.shields.io/static/v1?label=BY&message=Reemoz&color=pink)
 
```

Comment:
- [ ] Please your readme file is missing the `project title and description`, this kindly add it, as this will enable other users to quickly have an idea of what you project is about.